src/components/Onboarding/steps/SelectPIN/index.js

```diff
@@ -4,17 +4,36 @@ import React from 'react'
 
 import Box from 'components/base/Box'
 import TrackPage from 'analytics/TrackPage'
+import type { DeviceType } from 'reducers/onboarding'
 
+import { cleanDeviceName } from 'helpers/devices'
 import GrowScroll from 'components/base/GrowScroll'
 import { Title, FixedTopContainer } from '../../helperComponents'
 import OnboardingFooter from '../../OnboardingFooter'
 import SelectPINnano from './SelectPINnano'
 import SelectPINblue from './SelectPINblue'
+import SelectPINnanoX from './SelectPINnanoX'
 import SelectPINrestoreNano from './SelectPINrestoreNano'
 import SelectPINrestoreBlue from './SelectPINrestoreBlue'
-
 import type { StepProps } from '../..'
 
+const SelectPinSwitcher = ({
```

Comment:
I would just call this component `SelectPIN` because it's basically the generic version. the "Switcher" is implementation detail so don't need to be in the naming ;)

Reply:
Ok